pmm_psmdb-pbm_setup/Dockerfile

```diff
@@ -51,7 +51,12 @@ RUN set -ex; \
     else \
         yum -y install percona-backup-mongodb-${PBM_VERSION}.el8 ; \
     fi && \
-    if [ $PMM_VERSION == "latest" ]; then \
+    if [[ $PMM_VERSION == http* ]]; then \
+        yum -y install pmm2-client && \
+        curl -Lf -o /tmp/pmm2-client.tar.gz $PMM_VERSION && \
+        cd /tmp && tar -xvf pmm2-client.tar.gz --transform 's,^/*[^/]*,/pmm2-client,S' && \
+        cd /tmp/pmm2-client && ./install_tarball && cd ../ && rm -rf pmm2* ;\
```

Comment:
how the pmm client binaries are accessible per this install?
Please take a look at the following script(used in package testing): https://github.com/Percona-QA/package-testing/blob/PMM-7-install-fb-tarball/scripts/pmm2_client_install_tarball.sh

Reply:
In this setup we first install pmm package from repos which prepares all necessary links and unit files and second we untar the tarball over this installation so in the result we have fully working agent